src/dom/dom.ts

```diff
@@ -980,10 +1025,77 @@ export class dom {
         allTextNodes.forEach((node)=> {
            node.nodeValue = node.nodeValue.replace(/\s+([^\s]*)\s*$/, nbsp + '$1')
         })
+    }
 
+    /**
+     * Add an event listener to multiple events.
+     * Useful for when mouse and touch events should call the same listener.
+     */
+    static addEventListeners(
```

Comment:
I'm okay with adding this here but FYI, I think we should try to use DomWatcher as a general pattern.

Reply:
Removed. Used DomWatcher.